projects/ngx-sub-form/src/lib/deprecated/ngx-sub-form.component.spec.ts

```diff
@@ -481,6 +486,7 @@ describe(`NgxSubFormComponent`, () => {
       passwordRepeat: string;
     }
 
+    @Directive()
```

Comment:
Why are some marked as `@Directive()`... 1/2

Reply:
I made the switch to `@Component` because the new linter requires the Directive suffix on the classname, odd that it did not pick some up, it seems that the linter only looks at root level classes? They actually should be components, so making them all components would be the better option (I'd assumed that's what happened with the linter)